common/http/src/main/java/io/helidon/common/http/ContextualRegistry.java

```diff
@@ -69,79 +68,4 @@ static ContextualRegistry create() {
     static ContextualRegistry create(ContextualRegistry parent) {
         return new ListContextualRegistry(parent);
     }
-
-    /**
-     * Register a new instance.
-     *
-     * @param instance an instance to register
-     * @param <T>      a type of the registered instance
-     * @throws NullPointerException if the registered object is {@code null}
-     */
-    <T> void register(T instance);
-
-    /**
-     * Register a new instance using a provided supplier. The supplier is guarantied to be called at most once when it's
-     * requested by the {@link #get(Class)} method. The returned value is then registered and the supplier is never used again.
-     *
-     * @param type     a type of supplied instance
-     * @param supplier a supplier of the instance to register
-     * @param <T>      a type of supplied object
-     * @throws NullPointerException if the {@code type} or the {@code supplier} is {@code null}
-     */
-    <T> void supply(Class<T> type, Supplier<T> supplier);
```

Comment:
Ah. I see that you inherited this name and that this class is exposed via `Request`, so renaming would be a breaking change. Never mind.

Reply:
Yes, most of the context API is driven by this. We may revisit for version 2.0